src/utils.js

```diff
@@ -60,7 +60,7 @@ const combineBody = (previousBody, outputText) => {
     if (hasOutput(previousBody)) {
         return previousBody.replace(
             outputRegex,
-            outputText ? `<!-- output start -->\n${outputText}\n<!-- output end -->` : '',
+            outputText ? `\n<!-- output start -->\n${outputText}\n<!-- output end -->` : '',
```

Comment:
Can also be added to output end

Reply:
idk if it's necessary. because it would add uneven space between section. We already have blank line before the i18n title

Reply:
As you prfer